packages/models-library/src/models_library/function_services_catalog/services/demo_units.py

```diff
@@ -187,7 +187,7 @@ def build_input(schema):
             ),
             "radiation": build_input(
                 {
-                    "title": "Radiati0n", # it's not a typo
+                    "title": "Radiati0n",  # it's not a typo
```

Comment:
maybe a bit more than "it's not a typo" so that the next one understand why it's there?

Reply:
That is a valid point but I am not sure about the purpose. @odeimaiz can you give us some input?

Reply:
I'm pretty sure that you, @pcrespov, asked me to change it to ```Radiati0n```

Reply:
ok. i do not recall :-(

Reply:
I added a comment. It was related with an assumption in the tests